docs/content/connectors/db2-cdc.md

```diff
@@ -50,8 +50,8 @@ Notes
 
 ###  Not support BOOLEAN type in SQL Replication on Db2
 
-Only snapshots can be taken from tables with BOOLEAN type columns. Currently SQL Replication on Db2 does not support BOOLEAN, so Debezium can not perform CDC on those tables.
-Consider using a different type.
+Only snapshots can be taken from tables with BOOLEAN type columns. Currently, SQL Replication on Db2 does not support BOOLEAN, so Debezium can not perform CDC on those tables.
```

Comment:
This comma is useless here.

Reply:
Romve comma about `BOOLEAN` type not supported?